### DISCUSSION TITLE
v1.3.0

### DISCUSSION THREAD

**Discussion by JunkFood02** [JunkFood02/Seal#218] [Announcements]
## What's Changed
Bump up FFmpeg version to v5.1

Feature Update:
- Crop album cover to square when saving as audio #112 #182 
- Paste multiple URLs in custom command mode #191 

Bug Fix:
- SponsorBlock "Conversion failed" error #163 
- Aria2c is not usable with "Save as audio"
- App crashes if error occurs after switching language

Behavior Changes:
- Updates preferences on clicks with "Configure before download" dialog
- Move the directory of temporary files to internal storage of the app #211 

Add Vietnamese translations by Nguyễn Hoàng Minh

**Full Changelog**: https://github.com/JunkFood02/Seal/compare/v1.2.1...v1.3.0

<hr /><em>This discussion was created from the release <a href='https://github.com/JunkFood02/Seal/releases/tag/v1.3.0'>v1.3.0</a>.</em>

**Comment by Zinoujoker** [comment]
Good app keep up 👍☺️
